content/guides/validator/introduction.md

```diff
@@ -232,26 +232,43 @@ You can also manually construct the class instance and pass any arguments you li
 ```ts
 Route.post('users', async ({ request, response }) => {
   await request.validate(
-    new CreateUser({
+    new CreateUserValidator({
       countries: fetchAllowedCountries(),
       states: fetchAllowedStates()
     })
   )
 })
 ```
 
-Following is an example of using the validator classes outside of the HTTP request.
+Your validator class would look like the following:
 
 ```ts
-import { validate } from '@ioc:Adonis/Core/Validator'
-import CreateUser from 'App/Validators/CreateUserValidator'
+import { schema } from '@ioc:Adonis/Core/Validator'
+
+interface Country {
```

Comment:
Don't understand the point of adding these interfaces

Reply:
I added them for type hinting, as this seemed like what the example's type was looking for. I can remove the type hinting if you want though and just use `any`, if that's your preference, to make it more concise.